llvm/test/Transforms/MergeICmps/X86/alias-merge-blocks.ll

```diff
@@ -55,15 +55,15 @@ opeq1.exit:
   ret i1 %8
 }
 
-define zeroext i1 @part_sequent_eq_with_metadata() {
+define zeroext i1 @part_sequent_eq_with_metadata() !prof !2 {
 ; X86-LABEL: @part_sequent_eq_with_metadata(
 ; X86-NEXT:  bb01:
 ; X86-NEXT:    [[A:%.*]] = alloca [[S:%.*]], align 8
 ; X86-NEXT:    [[B:%.*]] = alloca [[S]], align 8
-; X86-NEXT:    [[TMP0:%.*]] = load i32, ptr [[A]], align 4, !range [[RNG0:![0-9]+]], !noundef !1
-; X86-NEXT:    [[TMP1:%.*]] = load i32, ptr [[B]], align 4, !range [[RNG0]], !noundef !1
+; X86-NEXT:    [[TMP0:%.*]] = load i32, ptr [[A]], align 4, !range [[RNG1:![0-9]+]], !noundef [[META2:![0-9]+]]
```

Comment:
why the rename `RNG0` -> `RNG1` ?

Reply:
UTC did that. Probably because we now pass --check-globals and some metadata is now apparent to UTC, and occurs earlier.